platforms/hyperledger-fabric/configuration/roles/create/ca-tools/peer/tasks/write.yaml

```diff
@@ -0,0 +1,36 @@
+##############################################################################################
+#  Copyright Accenture. All Rights Reserved.
+#
+#  SPDX-License-Identifier: Apache-2.0
+##############################################################################################
+
+---
+############################################################################################
+# This task copy the peer certificate to the path provided in network.yaml
+- name: Copy msp cacerts from auto-generated path to given path
+  shell: |
+    export CA_TOOL_CLI=$(KUBECONFIG={{ kubernetes.config_file }}  kubectl get po -n {{ component_name }} | grep "ca-tools" | awk '{print $1}')
+    KUBECONFIG={{ kubernetes.config_file }} kubectl cp {{ component_name }}/${CA_TOOL_CLI}:crypto-config/peerOrganizations/{{ component_name }}/peers/{{ peer.name }}.{{ component_name }}/msp/cacerts/ca-{{ component_name }}-{{ item.external_url_suffix }}-8443.pem {{ peer.certificate }}
+  loop: "{{ endorsers_peers }}"
+  loop_control:
+    loop_var: peer
+  when: 
+    - item.org_status == 'new'
+    - org_peer.peerstatus is not defined or org_peer.peerstatus == 'new'
+    - network.env.proxy != 'none'
+    - org_peer.name == peer.name
+
+############################################################################################
+# This task copy the peer certificate to the path provided in network.yaml (when proxy is none)
+- name: Copy msp cacerts from auto-generated path to given path
+  shell: |
```

Comment:
This task should be done by pulling the certificate from vault and not using `kubectl cp` cmd